dev-docs/modules/fledgeForGpt.md

```diff
@@ -2,172 +2,64 @@
 layout: page_v2
 page_type: module
 title: Module - fledgeForGpt
-description: how to use fledge with GPT
+description: how to use PAAPI with GPT
 module_code : fledgeForGpt
-display_name : Fledge for GPT
+display_name : Fledge (PAAPI) for GPT
 enable_download : true
 sidebarType : 1
 ---
 
-# Fledge (Protected Audience) for GPT Module
+# Fledge (PAAPI) for GPT Module
 
-This module allows Prebid.js to support FLEDGE by integrating it with GPT's [experimental FLEDGE
+This module allows Prebid.js to support PAAPI by integrating with GPT's [experimental
```

Comment:
It is not experimental (assuming the July release in GPT means we can drop that label). Emphasis mine.
https://developers.google.com/publisher-tag/release-notes#2023-07-24

>The [Protected Audience API](https://privacysandbox.com/news/protected-audience-api-our-new-name-for-fledge) (formerly known as FLEDGE) is [transitioning to general availability](https://privacysandbox.com/news/the-next-stages-of-privacy-sandbox-general-availability) with the July release of Chrome. _With this, component auction-related functionality in GPT is now considered stable._